src/main/java/liquibase/ext/hibernate/database/HibernateDatabase.java

```diff
@@ -36,13 +36,11 @@ public abstract class HibernateDatabase extends AbstractJdbcDatabase {
     protected Dialect dialect;
 
     private boolean indexesForForeignKeys = false;
-    public static final String DEFAULT_SCHEMA = "HIBERNATE";
+    public static final String DEFAULT_DEFAULT_CATALOG_NAME = "HIBERNATE";
```

Comment:
Is there any reason why these constants are named `DEFAULT_DEFAUL..`?

Reply:
I think my idea was that it is the default value for `getDefaultSchemaName`, so that's why there's two defaults. But I agree it's confusing, so I updated it.